doc/run/cli.md

```diff
@@ -77,13 +84,22 @@ Congratulations! Now you have successfully completed a session using SQLFlow syn
 
 ## Command-line Options
 
-|             Option                      | Description |
-|-----------------------------------------|-------------|
-| -e \<quoted-query-string\>              | Execute from command line without entering interactive mode. e.g. <br>`-e "SELECT * FROM iris.train TRAIN DNNClassifier..." `<br>does the same thing as the training example above.|
-| -f \<filename\>                         | Execute from file without entering interactive mode. e.g. <br>`-f ./my_sqlflow.sql`<br>does the same thing as<br>`< ./my_sqlflow.sql` and `cat ./my_sqlflow.sql \| REPL...` |
-| -model_dir \<local-directory\>          | Save model to a local directory. e.g. `-model_dir "./models/"` |
-| -datasource \<database-connection-url\> | Connect to the specified database. e.g. `-datasource "mysql://root:root@tcp(host.docker.internal:3306)/" ` |
-| -A                                      | No auto completion for `sqlflow_models`. This gives a quicker start |
+|             Option                      | Environment Variable   | Description |
+|-----------------------------------------|------------------------|-------------|
+| -sqlflow_server \<quoted-query-string\> |     SQLFLOW_SERVER     | Specify sqlflow server address, in `host:port` form, e.g. `-sqlflow_server "localhost:50051"` |
+| -e \<quoted-query-string\>              |                        | Execute from command line without entering interactive mode. e.g. <br>`-e "SELECT * FROM iris.train TRAIN DNNClassifier..." `<br>does the same thing as the training example above.|
+| -f \<filename\>                         |                        | Execute from file without entering interactive mode. e.g. <br>`-f ./my_sqlflow.sql`<br>does the same thing as<br>`< ./my_sqlflow.sql` and `cat ./my_sqlflow.sql \| sqlflow`. The special file `-` means read from standard input. |
+| -datasource \<database-connection-url\> |   SQLFLOW_DATASOURCE   | Connect to the specified database. e.g. `-datasource "mysql://root:root@tcp(localhost:3306)/" ` |
+| -A                                      |                        | No auto completion for `sqlflow_models`. This gives a quicker start. |
+
+## Environment Variable Config File Setup
+You can specify some of the options in a config file named `.sqlflow_env` under your home directory.  This process is **optional** but can be convenient if you use the same config intensively.  The content are exported as environment variables at run time. Be aware the config file is just a default setting, you can overwrite them via corresponding command-line options.  Currently supported variables are listed in `Environment Variable` column in above table. You can setup the file using the following `bash` code.  
+```bash
+cat <<EOF >~/.sqlflow_env
+SQLFLOW_SERVER=localhost:50051
+SQLFLOW_DATASOURCE=mysql://root:root@tcp(localhost:3306)/?maxAllowedPacket=0
```

Comment:
The command-line option corresponding to this configuration key `SQLFLOW_DATASOURCE` is `--datasource`.  How may I know the correspondence between key names and option names?

Reply:
Their correspondence are listed in the table of this section:
https://github.com/lhw362950217/sqlflow/blob/update_client_doc/doc/run/cli.md#command-line-options